pkg/controller/import-controller.go

```diff
@@ -23,13 +23,22 @@ const (
 	AnnImportPod = "cdi.kubevirt.io/storage.import.importPodName"
 	//LabelImportPvc is a pod label used to find the import pod that was created by the relevant PVC
 	LabelImportPvc = "cdi.kubevirt.io/storage.import.importPvcName"
+	//AnnCreateImage provides a const for our PVC createImage annotation
+	AnnCreateImage = "cdi.kubevirt.io/storage.import.createImage"
 )
 
 // ImportController represents a CDI Import Controller
 type ImportController struct {
 	Controller
 }
 
+type importPodEnvVar struct {
+	ep, secretName, imageSize string
+	createImage               bool
+}
+
+var podEnvVar importPodEnvVar
```

Comment:
See my comment in related (closed) PR:  https://github.com/kubevirt/containerized-data-importer/pull/502

Basically, I don't think it's safe to use a global variable here.

Reply:
Rebased to fix that.